mlflow/server/auth/__init__.py

```diff
@@ -897,6 +915,9 @@ def delete_registered_model_permission():
     return make_response({})
 
 
+csrf = CSRFProtect()
```

Comment:
Do we need to define this object globally? or can we put it inside the `create_app` method?

Reply:
Yeah, I saw the example, but I'm not sure if it's explicitly recommended or not.